op-e2e/actions/l2_challenger_test.go

```diff
@@ -530,23 +694,55 @@ interaction:
 		require.Equal(rt.t, types.ReceiptStatusSuccessful, rt.receipt.Status, "failed to progress interactive fault proof")
 	}
 
-	confirmReceipt, err := rt.miner.EthClient().TransactionReceipt(rt.t.Ctx(), rt.txHash)
-	require.NoError(rt.t, err)
-	require.Equal(rt.t, types.ReceiptStatusSuccessful, confirmReceipt.Status, "failed to confirm")
-
 	// check output is deleted
 	remoteOutput, err := rt.outputOracleContract.GetL2Output(nil, rt.outputIndex)
 	require.NoError(rt.t, err, "unable to get l2 output")
 	outputDeleted := val.IsOutputDeleted(remoteOutput.OutputRoot)
 	require.True(rt.t, outputDeleted, "invalid output is not deleted")
 
 	// check output submitter is changed to challenger
-	securityCouncilAddr, err := rt.colosseumContract.SECURITYCOUNCIL(nil)
-	require.NoError(rt.t, err)
-	require.Equal(rt.t, remoteOutput.Submitter, securityCouncilAddr)
+	require.Equal(rt.t, remoteOutput.Submitter, rt.challenger1.address)
+
+	if version == defaultValidatorSystemVersion {
+		// check pending bond amount before challenge is canceled
+		balance, err := rt.valPoolContract.BalanceOf(nil, rt.challenger2.address)
+		require.NoError(rt.t, err)
+		require.Equal(rt.t, balance.Int64(), defaultDepositAmount-rt.dp.DeployConfig.ValidatorPoolRequiredBondAmount.ToInt().Int64())
+	}
```

Comment:
Is there any reason that it does not check for `defaultValidatorSystemVersion+1` case?

Reply:
Because V2 does not have pending bond. If a challenge is canceled, there are nothing to be changed in the respect of assets.